### ISSUE TITLE
Textarea does not create new line when used in Datatable celleditor, instead completes edit mode

### ISSUE THREAD

**Issue by ArohanGit** [primefaces/primeng#10458]
**I'm submitting a bug report...**  (check one with "x")
```
[x] bug report => Search github for a similar issue or PR before submitting
[ ] feature request => Please check if request is not on the roadmap already https://github.com/primefaces/primeng/wiki/Roadmap
[ ] support request => Please do not submit support request here, instead see http://forum.primefaces.org/viewforum.php?f=35
```

**Plunkr Case (Bug Reports)**
https://stackblitz.com/edit/github-ytaja5

**Current behavior**
Textarea when used in Datatable Celleditor completes Edit Mode when Enter Or  ALT+Enter  Or  Shift+Enter key is pressed. 

**Expected behavior**
When ALT+ENTER Or Shift+ENTER key is presssed new line should be added in Textarea and Edit mode should continue

* **Angular version:** 11.X

* **PrimeNG version:** 11.2.0

* **Browser:** [all | Chrome XX | Firefox XX | IE XX  | Edge]
 
* **Language:** [all | TypeScript X.X | ES6/7 | ES5]

**Comment by mertsincan** [comment]
Hi,

So sorry for the delayed response! Improvements have been made to many components recently, both in terms of performance and enhancement. Therefore, this improvement may have been developed in another issue ticket without realizing it. You can check this in the documentation. If there is no improvement on this, can you reopen the issue so we can include it in our roadmap? 
Please don't forget to add your feedback as a comment after reopening the issue. These will be taken into account by us and will contribute to the development of this feature. Thanks a lot for your understanding!

Best Regards,

**Comment by ThibSCH** [comment]
Hi, 
I ran into the same issue and can confirm that this is still not fixed in the latest version (16.0 rc1).
As shown here : 
[demo](https://codesandbox.io/s/primeng-tree-table-edit-demo-forked-3yro2k?file=/src/app/demo/tree-table-edit-demo.html)
